src/nativeInterop/cinterop/PhoenixCrypto.def

```diff
@@ -9,5 +9,5 @@ libraryPaths.ios_x64 = PhoenixCrypto/build/Release-iphonesimulator
 libraryPaths.ios_arm64 = PhoenixCrypto/build/Release-iphoneos
 
 linkerOpts = -L/usr/lib/swift
-linkerOpts.ios_x64 = -ios_simulator_version_min 14.0.0 -L/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/swift/iphonesimulator/
+linkerOpts.ios_x64 = -L/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/swift/iphonesimulator/
```

Comment:
This option caused the build to fail (`ld: unknown options: -ios_simulator_version_min`

The core issue seems to be that support for this option has been removed with Xcode 15. So I removed it, and the build passes.

However some people [suggest using `-platform_version`](https://developer.apple.com/forums/thread/733536). @robbiehanson wdyt?

Reply:
Eventually used `platform_version` with min 15.0.0 in https://github.com/ACINQ/lightning-kmp/pull/565/commits/8f8624d7c41c78a83806fcf127eec9d2a355efbd.